docs/resources/database.md

```diff
@@ -1 +1,464 @@
-# :resources-database: Database
\ No newline at end of file
+# :resources-database: Database
+
+Database as a [Resource](/resources/) within DataOS is declared for creating a database within DataOS. It is treated as a first-class resource, encapsulating the fundamental unit for storing and organizing structured data using PostgreSQL. The classification of Databases as a resource type within the DataOS ecosystem allows for systematic management, provisioning, and utilization of databases in a standardized manner.
+
+<aside class=”callout”>
+
+ Database is a Workspace-level Resource type.
+
+</aside>
+
+
+<div class="grid cards" markdown>
+
+-   :material-card-bulleted-settings-outline:{ .lg .middle } **Key Features of Database**
+
+    ---
+
+    Understand the need of Database as a Resource in DataOS.
+
+    [:octicons-arrow-right-24: Key Features](#key-features-of-database)
+
+
+-   :material-list-box-outline:{ .lg .middle } **How to create a Database?**
+
+    ---
+
+    Learn how to create a database yaml specific to PostgreSQL.
+
+    [:octicons-arrow-right-24: Create a Database](#how-to-create-a-database?)
```

Comment:
This link doesn't work remove the ? from the end and check on your local machine